include/battle.h

```diff
@@ -326,9 +326,9 @@ struct BattleTv_Mon
 
 struct BattleTv
 {
-    struct BattleTv_Mon mon[2][PARTY_SIZE]; // [side][partyId]
-    struct BattleTv_Position pos[2][2]; // [side][flank]
-    struct BattleTv_Side side[2]; // [side]
+    struct BattleTv_Mon mon[NUM_BATTLE_SIDES][PARTY_SIZE];
+    struct BattleTv_Position pos[NUM_BATTLE_SIDES][2]; // [side][flank]
```

Comment:
If there was a `NUM_BATTLE_FLANKS` define then I suppose we could remove this comment too? (`NUM_BATTLE_FLANKS` may come in useful for your triple battle branch?)

Not really necessary though, was just a passing thought I had when scrolling through :)

Reply:
Num flanks sounds like the number of flanking battlers excluding some central battler. Maybe `NUM_BATTLERS_PER_SIDE`? I'd have to see where else that could be used to know if it'd be useful to have